clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -202,8 +202,7 @@ bool Sema::isKnownGoodSYCLDecl(const Decl *D) {
   if (const FunctionDecl *FD = dyn_cast<FunctionDecl>(D)) {
```

Comment:
Did something wacky go on here?  It looks like you applied clang-format to the entire file rather than just your changes.  We typically don't want you doing that, as it messes up history.

Reply:
When I run git-clang-format on my changes, it doesn't show me any errors. But buildbot check fails. So I tried clang-format, but it modifies the whole file :(

Reply:
I will try with --lines=<string> and revert the other changes